better-politics-mod/common/history/characters/brz - brazil.txt

```diff
@@ -0,0 +1,102 @@
+﻿CHARACTERS = {
```

Comment:
I noticed there's a lot of vanilla code in these character history files. I think we should only add BPM code and rename these files to "bpm_brz_brazil.txt" or something, unless we're explicitly overwriting vanilla code.